docs/source/contributing/setup.rst

```diff
@@ -30,15 +33,44 @@ After that, install dependencies and follow standard django procedures
 .. code-block:: bash
 
    pipenv install --dev
-   python3 manage.py migrate
-   python3 create_debug_users.py
+   pipenv run python3 manage.py migrate
+   pipenv run python3 scripts/create_debug_users.py
 
 
 Now you're all set! Try running the development server
 
 .. code-block:: bash
 
-   python3 manage.py runserver
+   pipenv run python3 manage.py runserver
 
 Head on over to `http://127.0.0.1:8000 <http://127.0.0.1:8000>`_, and login
 as ``admin`` and the password you just entered.
+
+Submissions
+~~~~~~~~~~~
+
+In order to actually create a submission, there are some more steps. First,
+you'll need to install `redis <https://redis.io/download>`_.
+
+You'll also need to run some scripts to emulate the sandboxing process that goes on in production.
+Run the following script::
+
+  pipenv run python3 scripts/create_wrappers.py
+
+After that, you'll want to start up the development server and create a course,
+and an assignment in the course. After saving the assignment, you can hit "Upload grader"
+to add a grader - the simplest example of a grader is located in ``scripts/sample_grader.py``.
+
+Finally, before creating a submission, you'll need to start the celery worker. This can be done
+by running the following command in a separate terminal::
+
+  pipenv run celery -A tin worker --loglevel=info
+
+Now you can try making a submission, and as long as your submission doesn't throw an error you
+should get a 100%! Congrats on your brand new 5.0 GPA!
+
+.. tip::
+
+  If you're on a Unix-like system, you can use the ``&`` operator to run the celery worker in the background::
+
+    pipenv run celery -A tin worker --loglevel=info & pipenv run python3 manage.py runserver
```

Comment:
Does this work? I thought you needed a `&&` or `;` or something like that...

Reply:
It does work (I tested on zsh and bash). It effectively parses as
```
cmd a &
cmd b
```